charts/fluent-operator/values.yaml

```diff
@@ -0,0 +1,176 @@
+# Default values for fluentbit-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Set this to containerd or crio if you want to collect CRI format logs
+containerRuntime: docker
+Kubernetes: false
```

Comment:
I think we need a few more comments in this file to explain some of the settings, like this one.